integration-tests/src/server.js

```diff
@@ -0,0 +1,64 @@
+const express = require('express')
+const http = require('http')
+const puppeteer = require('puppeteer')
+const _ = require('lodash/fp')
+const { promiseHandler, Response } = require('./utils')
+const { testFindWorkflow } = require('../tests/find-workflow')
+const { testImportCohortData } = require('../tests/import-cohort-data')
+const { testImportDockstoreWorkflow } = require('../tests/import-dockstore-workflow')
+const { testRegisterUser } = require('../tests/register-user')
+const { testRunWorkflow } = require('../tests/run-workflow')
+const { defaultTimeout } = require('../utils/integration-helpers')
+const { delay, withScreenshot } = require('../utils/integration-utils')
+const envs = require('../utils/terra-envs')
+
+
+const app = express()
+const server = http.createServer(app)
+server.setTimeout(defaultTimeout + 60 * 1000) // allow an extra minute for teardown
+
+const getBrowser = _.once(() => puppeteer.launch())
+const getContext = async () => {
+  const browser = await getBrowser()
+  return browser.createIncognitoBrowserContext()
+}
+
+const testTimeout = async (context, timeout) => {
+  await delay(timeout)
+  context.close()
```

Comment:
I'll experiment. There's some trickiness to making sure the context is closed at the right time. In truth, this isn't quite the right time because we don't currently get a screenshot when the test times out like this.